lib/authzed/api/v1alpha1/client.rb

```diff
@@ -0,0 +1,26 @@
+require 'grpc'
+
+module Authzed
+  module Api
+    module V1alpha1
+      class Client
+
+        attr_reader :schema_service
+
+        def initialize(target:, token:, credentials: nil, options: {}, timeout: nil)
+          bearer = ::Authzed::GrpcUtil::BearerToken.new(token: token)
+          creds = credentials || GRPC::Core::ChannelCredentials.new
+
+          @schema_service = Authzed::Api::V1alpha1::SchemaService::Service::Stub.new(
+            target,
+            creds,
+            timeout: timeout,
+            interceptors: [bearer],
```

Comment:
Note: using interceptor instead of credentials to implement bearer token which is different than python client

Reply:
Do we think it could ever make sense to pass the token as an option? This is how most Go does it, for example.